### PR TITLE
Fix www.geartics.com Ad,Social

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 https://www.geartics.com/device-rankings/mousepad

### Add your comment and screenshots

<details><summary>screenshot</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: JapaneseFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -641,7 +641,7 @@ gadget2ch.com#?#.posts-inner:contains(/スポンサードリンク|関連コン
 libae.jp#?#.post > div.pc_display > div[id^="logly"]:upward(1)
 2ch-c.net#?#.nend_text_ad:upward(li)
 hibiki-radio.jp#?#.ng-isolate-scope > li.ng-scope ins.adsbygoogle:upward(li)
-geartics.com#?#.mb-3.text-center:contains(スポンサーリンク)
+geartics.com#?#.my-8:has(> div.ads-double > div > div > ins[data-ad-client])
```
Comment (Yuki2718) [inline JapaneseFilter/sections/css_extended.txt:644]: `#?#.my-8:has(> div.ads-double)` seems to be enough, also add `geartics.com#?#.white:has(+ .my-8 > div.ads-double) > hr:last-child`.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -18,6 +18,7 @@ gitpod.io#?##main-content div.flex > div.flex:has(> h2:contains(Share))
 tudn.com,univision.com#?#div[class]:contains(/^Comparte$/)
 wallscloud.net#?#.container-right > div.rside-box:has(> div.share-wall)
 shindanmaker.com#?##main > div.pb-5:has(> a.twitter-share-button)
+geartics.com#?#.mt-10 > div.text-center:contains(このページをシェア)
```
Comment (Yuki2718) [inline SocialFilter/sections/css_extended.txt:21]: `#?#.mt-10 > div.text-center:has(+ div.share-buttons)`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/css_extended.txt (hunk 1 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/css_extended.txt:644]: I see. 
I will fix it.

File: SocialFilter/sections/css_extended.txt (hunk 2 above)
Comment (SKEIDs) [reply SocialFilter/sections/css_extended.txt:21]: I will fix

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**